mParticle-Leanplum.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
     s.ios.deployment_target = "9.0"
     s.ios.source_files      = 'mParticle-Leanplum/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 8.0'
-    s.ios.dependency 'Leanplum-iOS-SDK', '~> 4.1'
+    s.ios.dependency 'Leanplum-iOS-SDK', '~> 6.0'
```

Comment:
Any reason not to align on 6.1 here?

Reply:
@jasonmgeorge It'll automatically take anything up to 6.x with this. I could put 6.1 to make that the minimum but I think keeping it .0 helps people remember the "up to next major version" bit better

Reply:
It would make sense to be aligned with the others, though. No?

Reply:
@via-guy agreed, we should either change this one to 6.1 or change the others to 6.0 and 6.0.0 respectively